assets/js/src/form_editor/rich_text/font_selection_format.tsx

```diff
@@ -0,0 +1,52 @@
+import React from 'react';
+import { applyFormat } from '@wordpress/rich-text';
+import MailPoet from 'mailpoet';
+import { Fill } from '@wordpress/components';
+
+import FontFamilySettings from '../components/font_family_settings';
+
+const name = 'mailpoet-form/font-selection';
+const title = 'Font Selection';
+
+const settings = {
+  name,
+  title,
+  tagName: 'span',
+  className: 'has-font',
```

Comment:
This is probably too common class name. Let's rather use `mailpoet-has-font`. I think we should also add a fallback font family on frontend using this class in public.js